### ISSUE TITLE
RFC: automate the release process

### ISSUE THREAD

**Issue by ylatuya** [fluendo/fluster#135]
Fluster hasn't had a release yet, which we will need to start doing to package it for pypi and Debian.

The proposal is the use to use [Conventional Commit Messages](https://www.conventionalcommits.org/en/v1.0.0/)  in combination with the following github action https://github.com/google-github-actions/release-please-action to automate all the release process.

**Comment by rsanchez87** [comment]
Hello @ylatuya , we have already an automated process to create a release version from the CI, you can see these PRs:

https://github.com/fluendo/fluster/pull/210
https://github.com/fluendo/fluster/pull/213
https://github.com/fluendo/fluster/pull/230

or the next file (release.yml):

https://github.com/fluendo/fluster/blob/master/.github/workflows/release.yml

**Comment by ylatuya** [comment]
I don't think this is an automated release process since you still have to manage version bumps and the changelog generation manually. By using semantic commits and the Google github action I mentioned or https://python-semantic-release.readthedocs.io/en/latest/index.html all the process is automated from the commit messages, including versions bump, changelog generation, etc...

**Comment by rsanchez87** [comment]
> I don't think this is an automated release process since you still have to manage version bumps and the changelog generation manually. By using semantic commits and the Google github action I mentioned or https://python-semantic-release.readthedocs.io/en/latest/index.html all the process is automated from the commit messages, including versions bump, changelog generation, etc...

PR created:
https://github.com/fluendo/fluster/pull/237

Thanks!

**Comment by ylatuya** [comment]
I think this issue can be closed now that #237  has been merged
